llvm/test/CodeGen/RISCV/rvv/vselect-fp.ll

```diff
@@ -519,6 +519,7 @@ define void @vselect_legalize_regression(<vscale x 16 x double> %a, <vscale x 16
 ; CHECK-NEXT:    vmv.v.i v24, 0
 ; CHECK-NEXT:    vmerge.vvm v16, v24, v16, v0
```

Comment:
Any idea why we're not turning this into a vmerge.vim with a negated mask?  That would radically reduce register pressure and work, and be net equal in terms of instructions.